lib/delayed/backend/worker.rb

```diff
@@ -16,7 +16,7 @@ def configure
                       end
         self.delay = config.delay_seconds || 0
         self.timeout = config.visibility_timeout || 5.minutes
-        self.expires_in = config.message_retention_period || 4.days
+        self.expires_in = config.message_retention_period || 345600 # 4 days
```

Comment:
I can imagine something along the lines of the ruby 2.4 changes where they unified the integer classes being at play. Just curious here, it felt odd that one is done "manually" while a line above we still use the "human friendly way". I can already see someone a year from now changing this back to `4.days` because `5.minutes` sets an example.